### ISSUE TITLE
Keyboard navigation doesn't work on "Tree Table" example

### ISSUE THREAD

**Issue by rutinerad** [Addepar/ember-table#80]
When I go to the project demos (at http://addepar.github.io/ember-table/) and click a row in the _Tree Table_ example, the keyboard expand/collapse (left/right arrow button) doesn't work. The icon changes, but nothing else happens.

I am running the latest version of Chrome.
